deploy/kubernetes/base/controller/cluster_setup.yaml

```diff
@@ -1,3 +1,10 @@
+kind: Namespace
```

Comment:
I'm curious why none of our tests were failing without this?

Reply:
cc @verult

Reply:
The installation script also creates a secret in the namespace using the JSON key provided by the user, which doesn't have a retry loop, so if the namespace hasn't been created yet the deployment would be broken.

Reply:
Got it.